hack/dockerized

```diff
@@ -74,14 +74,14 @@ _rsync() {
 rm -f ${KUBEVIRT_DIR}/.glide*.hash
 
 # Copy kubevirt into the persistent docker volume
-_rsync --delete --exclude 'cluster/**/.kubectl' --exclude 'cluster/**/.oc' --exclude 'cluster/**/.kubeconfig' --exclude "_out" --exclude ".vagrant" ${KUBEVIRT_DIR}/ "rsync://root@127.0.0.1:${RSYNCD_PORT}/build"
+_rsync --delete --exclude 'bazel-*' --exclude 'cluster/**/.kubectl' --exclude 'cluster/**/.oc' --exclude 'cluster/**/.kubeconfig' --exclude "_out" --exclude ".vagrant" ${KUBEVIRT_DIR}/ "rsync://root@127.0.0.1:${RSYNCD_PORT}/build"
```

Comment:
forgot one thing: not 100% if we should hide the bazel-* directories. People might use bazel directly, and then we have 2 bazel caches? Might cause confusion.

Reply:
It useless to copy bazel directories, because of it only soft links on the cache under `~/.cache/bazel`

Reply:
oh, I see